pybuildkite/pipelines.py

```diff
@@ -98,6 +122,7 @@ def update_pipeline(
         name: str = None,
         provider_settings: dict = None,
         repository: str = None,
+        configuration: str = None,
         steps: dict = None,
```

Comment:
Do we need to catch and throw an error if somebody tried to use both of these arguments? We could throw a custom Exception and explain which one they should use.

Reply:
Good call, let me add that